scripts/mina-local-network/mina-local-network.sh

```diff
@@ -145,6 +148,8 @@ exec-daemon() {
     -log-json \
     -log-level ${LOG_LEVEL} \
     -file-log-level ${FILE_LOG_LEVEL} \
+    -precomputed-blocks-file ${FOLDER}/precomputed_blocks.log \
```

Comment:
Will the `$FOLDER` be available here from invocation point at `spawn-node()`? If it will then cool.

Reply:
Yes, i tested it locally and it is nicely propagated from upper level to here pointing to node root folder

Reply:
Cool, thanks!